src/reducers/index.js

```diff
@@ -19,8 +19,15 @@ const reducer = (state = getDefaultState(), action) => {
         roomId: action.roomId
       });
     case 'GET_ROOMS':
```

Comment:
Related this [comment](https://github.com/duyoji/oni-on-cheese/pull/92/files#r158680584), Maybe this action typeis no longer used.